lib/perl/Genome/Site/TGI/Extension/Sys.pm

```diff
@@ -78,10 +78,6 @@ sub bsub_and_wait_for_completion {
     ($special_args, %bsub_args) = _bsub_and_wait_for_completion__validate_args(%bsub_args);
     my($commands, $on_submit_cb, $on_complete_cb) = @$special_args{'commands','on_submit','on_complete'};
```

Comment:
If we're removing the `on_complete` callback support, we should clean this and the `__validate_args` subroutines up to no longer mention it as well.